config/authentication/auth_config.yml

```diff
@@ -0,0 +1,24 @@
+development:
+  omniauth_providers: [developer]
```

Comment:
**nitpick (blocking):** can we set this to an empty list in `development` and after that is updated can this file be added to .gitignore to not be tracked so that if a person wants to override locally for development it doesn't show up as changes to commit.

Reply:
changed in https://github.com/phac-nml/irida-next/pull/420/commits/151277cba85abe91ee711401239daa74f82385e1